tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -0,0 +1,283 @@
+@api @skipOnOcV10
```

Comment:
these are the local API tests inside oCIS, do we ever run these tests using oC10 backend?

Reply:
No, the `skipOnOcV10` is not relevant. All the feature files have this tag in them - it is a bit of "history". It does not do any harm, and if someone does try to run against OcV10 then the tests will get skipped.